fileutils.go

```diff
@@ -139,19 +139,16 @@ func CopyDirectory(source string, dest string) error {
 			return nil
 		}
 
-		// Copy the file.
-		if err := CopyFile(path, filepath.Join(dest, relPath)); err != nil {
-			return err
-		}
-
-		return nil
+		return CopyFile(path, filepath.Join(dest, relPath))
```

Comment:
Here I just noted that checking for the error was a bit redundant, if there's was a reason for that just let me know and I'll send without this

Reply:
:+1: